exercises/proverb/package.yaml

```diff
@@ -0,0 +1,17 @@
+name: proverb
+version: 1.1.1.2
```

Comment:
since 1.1.1 doesn't exist in https://github.com/exercism/problem-specifications/blob/master/exercises/proverb/canonical-data.json#L3, the first thee digits should be 1.1.0 instead, which is what exists in https://github.com/exercism/problem-specifications/blob/master/exercises/proverb/canonical-data.json#L3 .  The last digit should be 1.

so, 1.1.0.1 please.

documented in https://github.com/exercism/haskell/blob/master/README.md#exercise-versioning